src/tokenizer/types.js

```diff
@@ -61,7 +61,7 @@ export class TokenType {
     this.isAssign = !!conf.isAssign;
     this.prefix = !!conf.prefix;
     this.postfix = !!conf.postfix;
-    this.binop = conf.binop || null;
+    this.binop = conf.binop === 0 ? 0 : conf.binop || null;
```

Comment:
This will be nice with ?? :)

Reply:
Yes, should be easy to add/use here (would be good to find this kind of code on github/codebase and then do a codemod)